pkgs/development/python-modules/w3lib/default.nix

```
@@ -25,8 +26,10 @@ buildPythonPackage rec {
     "w3lib"
   ];
 
-  disabledTests = [
-    "test_add_or_replace_parameter"
+  disabledTests = lib.optional (pythonAtLeast "3.11") [
```

This should be `lib.optionals` since the second argument is a list
```suggestion
  disabledTests = lib.optionals (pythonAtLeast "3.11") [
```
Besides that. LGTM